tools/compare-wp-performance/new/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "@compare-wp-performance/new",
+	"private": true,
+	"description": "Comparing performance between two WordPress core releases with benchmarking",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/swissspidy/compare-wp-performance"
+	},
+	"author": "swissspidy",
+	"license": "Apache-2.0",
+	"bugs": {
+		"url": "https://github.com/swissspidy/compare-wp-performance/issues"
+	},
+	"homepage": "https://github.com/swissspidy/compare-wp-performance#readme",
+	"type": "module",
+	"devDependencies": {
+		"@wordpress/env": "^8.4.0",
```

Comment:
Couldn't we alternatively use an older version that doesn't have the bug and thus doesn't require the patch and extra `package.json` workaround?

Reply:
I will need to check what‘s possible / where the bug was introduced and if there are features we can live wirhout or not.

Reply:
The bug has since been resolved, so we can use a newer version without a patch, but it still requires 2 package.json files. Or we leverage a global install.